pkg/config/db.go

```diff
@@ -117,13 +102,13 @@ func (c *DatabaseConfig) ConnectionStringWithName(name string, withSSL bool) str
 	var cmd string
 	if withSSL {
 		cmd = fmt.Sprintf(
-			"host=%s port=%d user=%s password='%s' dbname=%s sslmode=%s sslrootcert=%s",
-			c.Host, c.Port, c.Username, c.Password, name, c.SSLMode, c.RootCertFile,
+			"host=%s port=%d user=%s dbname=%s sslmode=%s sslrootcert=%s",
```

Comment:
```suggestion
			"host=%s port=%d user=%s dbname=%s sslmode=%s sslrootcert='%s'",
```

it is possible for someone to configure an empty ssl root cert (in which case the system default will be used).
If we don't quote it, errors like https://github.com/openshift-online/maestro/pull/231/files#r1888013192 happens